lib/rpminstall.cc

```diff
@@ -667,15 +668,23 @@ int rpmInstall(rpmts ts, struct rpmInstallArguments_s * ia, ARGV_t fileArgv)
 	rpmcliProgressState = 0;
 	rpmcliProgressTotal = 0;
 	rpmcliProgressCurrent = 0;
+
+	tsflags = rpmtsFlags(ts);
+	otsflags = rpmtsSetFlags(ts, (tsflags | RPMTRANS_FLAG_NOPLUGINS));
```

Comment:
Using NOPLUGINS is fine I suppose, but it needs to be set inside rpmInstallSourcePackage() as that's the lowest level public API we have for that.

Reply:
Ah right, haven't considered *that* at all. Sigh :smile: